.github/workflows/backwards.yml

```diff
@@ -16,6 +16,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: Merge Environments
+      shell: bash -l {0}
+      run: |
+        awk 'NR>3{ print $0 }' .ci_support/environment-notebooks.yml >> .ci_support/environment.yml
     - uses: conda-incubator/setup-miniconda@v2
```

Comment:
Is this change related to the windows tests?

Reply:
No, it is the same change as it was done for the notebook testing. In the tests the environment resolution took more than 30 minutes and resulted in lots of conflicts.

Reply:
Ok, right - now I see. Thanks for the explanation.